.eslintrc.js

```diff
@@ -8,6 +8,9 @@
 module.exports = {
   extends: ['@newrelic', 'plugin:jsdoc/recommended'],
   plugins: ['jsdoc'],
+  parserOptions: {
+    ecmaVersion: '2020'
```

Comment:
I think ideally when we bump allowed versions based on what's allowed (deprecating older node versions), we update the shared definition so all projects are consistent. Fine to land this sort of thing in the project but might follow-up with updating the shared eslint module and then removing here.

Also, I'm not sure what was needed to bump that version but calling out that this will probably allow "spread parameters after optional chaining" which is not available in Node 14. So there's a small risk of introducing something that won't work. Hopefully that would get caught by tests but noting we have been burned at least once since I've been involved with the Node team.

Reply:
The reason I introduced this is that doing `await import()` without it caused a parsing error on my unit test file, causing eslint to fail completely on the file (which would then fail CI). Fixing that is not worth the risk of potentially allowing introduction of bugs for v14 IMO, so what do you think about undoing this change and then adding my ESM unit test to `.eslintignore`?

Reply:
We have at times had to have a config specific to the tests folder to bump the ecmascript version earlier than the code base for testing newer features. For example, async-await while still supporting very old node versions.

It looks like we still have a file in there, so I might just bump that one to start: `test/.eslintrc.js`.

Reply:
for future posterity 16.3.0 is when es2020 spec is 100% implemented: see https://node.green/